index.html

```diff
@@ -646,25 +694,32 @@ <h2>
           <a><code>OrientationLockType</code></a> enum.
         </p>
         <p>
-          A <a>document</a>'s <dfn>orientation lock</dfn> is the orientation
-          lock that applies on its <a>top-level browsing context</a>. An
-          orientation lock is an unordered set of <a>OrientationType</a>.
+          The <dfn>[[\orientationLock]]</dfn> internal slot represents the
+          <a>document</a>'s' <a>orientation lock</a>.
+        </p>
+        <p>
+          An <dfn>orientation lock</dfn> is in place when the screen has
+          successfully been locked to a specific orientation.
         </p>
       </section>
       <section>
         <h2>
           Default orientation
         </h2>
         <p>
-          A <a>document</a>'s <dfn>default orientation</dfn> is the set of
-          orientations to which the screen orientation is locked when it is not
-          explicitly locked by this API or any other means.
+          The <a>[[\defaultOrientation]]</a> internal slot represents the
+          <a>document</a>'s <a>default orientation</a>.
         </p>
-        <p data-link-for="OrientationLockType">
-          For the perspective of a <a>document</a>, locking to the <a>default
+        <p>
+          The <dfn>default orientation</dfn> is the set of orientations to
+          which the screen is locked when there is no current <a>orientation
+          lock</a>.
+        </p>
+        <p>
+          From the perspective of a <a>document</a>, locking to the <a>default
           orientation</a> is equivalent to unlocking because it means that it
-          no longer has a lock applied. However, it does not mean that the
-          <a>default orientation</a> has to be <a>any</a>.
+          no longer has a lock applied. However, this does not mean that the
+          <a>[[\defaultOrientation]]</a> has to be <a>any</a>.
```

Comment:
remember, we said above that [[defaultOrientation]] is a set of values... so this should say something like "only contains the item "any"". 

You might need to check in the infra spec if sets contain items, values, or elements. I can't ever remember :(.

Reply:
You were right - it is item 😀

Reply:
Heh, lucky guess.